ELSpace/Commons/Extensions/UIAlertController+Building.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+extension UIAlertController {
+
+    convenience init(title: String?, customView: UIView, fallbackMessage: String?, preferredStyle: UIAlertController.Style) {
+
+        let marker = "__CUSTOM_CONTENT_MARKER__"
+        self.init(title: title, message: marker, preferredStyle: preferredStyle)
+
+        if let customContentPlaceholder = self.view.findLabel(withText: marker),
+            let customContainer =  customContentPlaceholder.superview {
+            customContainer.addSubview(customView)
```

Comment:
[Apple docs](https://developer.apple.com/documentation/uikit/uialertcontroller) says that
> The UIAlertController class is intended to be used as-is and does not support subclassing. The view hierarchy for this class is private and must not be modified.

Do we really need to do it this way? :)

Reply:
The implementation is not perfect :) But it has really nice usage .. maybe we could together figure sth out that requires only little work and is more reliable ?

Reply:
Something similar was done in PGG. You can ask @mateuszszklarek for his opinion